### PR TITLE
Tidy up the codebase to meet our software standards.

### PR DESCRIPTION
**Pull request by hsorby**
Addresses issue #116.

### CODE REVIEW

File: src/CMakeLists.txt [hunk 1]
```diff
@@ -28,7 +27,6 @@ set(LIBCELLML_LIBRARY_VERSION_STRING "${libCellML_VERSION_MAJOR}.${libCellML_VER
 set(LIBCELLML_MATHML_DTD_LOCATION "${CMAKE_CURRENT_SOURCE_DIR}/dtds/mathml2/mathml2.dtd")
 
 set(LIBCELLML_EXPORTDEFINITIONS_H "${CMAKE_CURRENT_BINARY_DIR}/api/libcellml/exportdefinitions.h")
-set_source_files_properties(${LIBCELLML_EXPORTDEFINITIONS_H} PROPERTIES GENERATED TRUE)
```
Comment (nickerso) [inline src/CMakeLists.txt:31]: just curious why we don't want to set this?

**Comment by nickerso** [comment]
I assume the reference to #116 implies there is more to come on this pull request?

File: src/CMakeLists.txt (hunk 1 above)
Comment (hsorby) [inline src/CMakeLists.txt:31]: It's implicit in the generate_export_header command.

**Comment by hsorby** [comment]
I was thinking either lump it all together or do separate pr's against the one issue, thoughts?

**Comment by nickerso** [comment]
separate pr's against the one issue makes it easier for different people to jump in, but would be good to specify which aspect of #116 is being addressed in each PR to make it clear when it is useful to start reviewing and merging...

**Comment by hsorby** [comment]
This pull request is for small general changes associated with cleaning up the code and not part of the listed items in issue #116.

**Pull request merged**